assets/css/main.scss

```diff
@@ -34,6 +34,33 @@ body {
   }
 }
 
+.csh-hacks-wrapper{
+  padding-top: 10px;
+  padding-bottom: 10px;
+  color: rgba(217, 184, 206, 1.00);
+  background: rgba(37, 2, 26, 1.0) repeat top/2.5em url("../csh_tilted.png"); 
+
+  & h2{
+    color: white;
+  }
+
+  &.padded {
+    padding-top: 30px;
+    padding-bottom: 30px;
+  }
+
+  & img{
+    width: 100%;
+    padding-top: 25px;
+  }
+
+  @media screen and (max-width: 992px){
+    padding-top: 0px;
+  }
+}
+
+
+
```

Comment:
Add this CSS to a different file, i.e. `_sass/_hackathon.sass`, and import it in `main.scss`.

Reply:
In addition, once you've moved the CSS, we already have helper classes to add padding to containers, see [here](https://github.com/ComputerScienceHouse/CSHPublicSite/blob/develop/assets/css/main.scss#L22).

Reply:
Looking through it, you don't even use the `.padded`, please remove it.

Reply:
going to move the CSS for the hackthon section to a separate file but out of curiosity why do we do this?